### PR TITLE
Consolidate sanity/screenshot/adversarial cloudbuild with domain support

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: build/ci/cloudbuild.website_testing.yaml [hunk 1]
```diff
@@ -13,16 +13,38 @@
 # limitations under the License.
 
 steps:
+  # Run Website sanity
+  - id: website_adversarial
```
Comment (keyurva) [inline build/ci/cloudbuild.website_testing.yaml:17]: I'm wondering if it's worthwhile keeping these cloudbuild files around so an individual test can be triggered manually?

**Review by keyurva — APPROVED** [review]
Thanks Bo!

File: build/ci/cloudbuild.website_testing.yaml (hunk 1 above)
Comment (shifucun) [reply build/ci/cloudbuild.website_testing.yaml:17]: Now we can still trigger those, but they would all run, would that work? I am bit worried maintaining several copies of very similar files..

File: run_website_sanity.sh [hunk 2]
```diff
@@ -15,21 +15,18 @@
...
+# DC Instance domain like: "dev.datacommons.org", "datacommons.org"
+domain=$1
+echo "Domain: $domain"
+
 export FLASK_ENV=webdriver
 export GOOGLE_CLOUD_PROJECT=datcom-website-dev
 
 python3 -m venv .env
 source .env/bin/activate
 pip3 install -r server/requirements.txt
```
Comment (pradh) [inline run_website_sanity.sh:27]: nit: maybe add a check that domain doesn't start with `http`?

**Review by pradh — COMMENTED** [review]
Thanks!

**Pull request merged**